src/changelog.ts

```diff
@@ -283,7 +324,7 @@ class Changelog {
    * @param {Version} version - The version of the release being retrieved.
    * @returns {ReleaseChanges} The changes included in the given released.
    */
-  getReleaseChanges(version) {
+  getReleaseChanges(version: string) {
```

Comment:
This should be `Version` as well
```suggestion
  getReleaseChanges(version: Version) {
```